common/changes/@microsoft/rush-stack-compiler-3.3/ianc-change-package-export_2020-05-28-04-51.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush-stack-compiler-3.3",
+      "comment": "Change the way the typescript, tslint, and api-extractor packages are exported.",
```

Comment:
This could be more clear. The current wording makes it sound like an API change.

Maybe "Improve performance by lazy-loading expensive package exports"?

Reply:
This actually is an API change.